### PR TITLE
Added policies and ToC footer links

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Adds footer links for
- cookie policy
- privacy policy
- terms and conditions

Screenshot 2023-01-20 at 14 27 14

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , great job!
However, seems the ToC may mean 'Table of content', not terms. 
Screenshot_2

Maybe we should write `CoW Terms` instead? 
image

**Comment by fairlighteth** [comment]
I would remove the "CoW" part from the labels.

File: const/menu.ts [hunk 1]
```diff
@@ -30,6 +30,10 @@ export const FOOTER_LINK_GROUPS: FooterLinkGroup[] = [
       { label: 'CoW Protocol', url: '/#about' },
       { label: 'CoW Swap', url: 'https://swap.cow.fi/#/about', type: 'external' },
       { label: 'CoW Swap FAQ', url: 'https://swap.cow.fi/#/faq', type: 'external' },
+      { label: 'CoW ToC', url: 'https://swap.cow.fi/#/terms-and-conditions', type: 'external' },
```
Comment (alfetopito) [inline const/menu.ts:33]: Following Elena's comment what we probably want here is

```suggestion
      { label: 'CoW T&C', url: 'https://swap.cow.fi/#/terms-and-conditions', type: 'external' },
```

Comment (alfetopito) [reply const/menu.ts:33]: Or use the longer version (`Terms and Conditions`) to reduce potential ambiguity

**Review by anxolin — APPROVED** [review]

File: const/menu.ts (hunk 1 above)
Comment (nenadV91) [reply const/menu.ts]: Good point, went with full `Terms and Conditions`

**Comment by nenadV91** [comment]
@elena-zh Updated

**Review by elena-zh — APPROVED** [review]
LGTM!

**Comment by anxolin** [comment]
@nenadV91 make sure to notify the commander either this week or next week (maybe is to small of a change to do it this week)

**Comment by nenadV91** [comment]
@anxolin I don't think we want to merge this until we move Policy and TOC pages to cow.fi which might take a bit longer. I'm not working on that currently.

**Review by alfetopito — APPROVED** [review]

**Pull request merged**